bundle

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+usage() { echo "Usage $0 [-n name] -o output_path -c channel" 1>&2; exit 1; }
+
+while getopts "n:o:c:" arg; do
+    case "${arg}" in
+        n)
+            name=${OPTARG}
+            ;;
+        o)
+            output=${OPTARG}
+            ;;
+        c)
+            channel=${OPTARG}
+            ;;
+        *)
+            echo "broke ${arg} - ${OPTARG}"
```

Comment:
```suggestion
```
getopts will tell you what broke -- and make `arg` and `OPTARG` not very useful in the `echo`:
```bash
$ bash bundle -f ohno
bundle: illegal option -- f
broke ? - 
Usage bundle [-n name] -o output_path -c channel
```